thumbor/config.py

```diff
@@ -131,6 +131,7 @@
 Config.define('ALLOW_OLD_URLS', True, 'Indicates if encrypted (old style) URLs should be allowed', 'Security')
 Config.define('ENABLE_ETAGS', True, 'Enables automatically generated etags', 'HTTP')
 Config.define('MAX_ID_LENGTH', 32, 'Set maximum id length for images when stored', 'Storage')
+Config.define('GC_INTERVAL', 60, 'Set garbage collection interval in seconds', 'Performance')
```

Comment:
I already updated the default here https://github.com/thumbor/thumbor/pull/905/files#diff-cbd5a27df49cf19d11956cefe1692c6bR134 but for some reason it won't release your "review". I'm going to dismiss it but of course I made the change and appreciate the review : )